dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/JSONToolTest.java

```diff
@@ -0,0 +1,321 @@
+package com.dotcms.rendering.velocity.viewtools;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test {@link JSONTool}
+ * @author jsanca
+ */
+public class JSONToolTest extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315721)